addons/udes_security/models/mail_thread.py

```diff
@@ -0,0 +1,73 @@
+from odoo import models, fields, _, api
+from odoo.exceptions import AccessError, UserError
+
+from html.parser import HTMLParser
+import re
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    def _message_create(self, values_list):
+        if not isinstance(values_list, (list)):
+            values_list = [values_list]
+
+        for vals in values_list:
+            body_content = vals.get('body', False)
+            #NOTE: message_type of email is also 'comment'! Check email specific fields
+            is_email = vals.get('message_type', False) == 'email' or \
+                       vals.get('mail_server_id', False)
+            #Strip html tags from 'Send message', 'Log note', 'Conversations'; except email
+            if body_content and not is_email:
+                vals['body'] = self._strip_html_tags(body_content)
+
+        return super(MailThread, self)._message_create(values_list)
```

Comment:
Looks like we have test failures because we also add messages into mail.thread from other parts of the system. Stripping HTML at this level could mean that we're also losing that functionality (as per the unit tests). Can we strip the HTML closer in the stack trace to "Conversations", "Log note", "Send message" etc to avoid any undesirable impacts?